src/utils/formatting.ts

```diff
@@ -3,6 +3,50 @@ import locale from 'date-fns/locale/en-US';
 
 import { Person } from 'src/types/domain';
 
+export const formatLongPublishDateString = function (
+  dateString: string
+): string {
+  const date = new Date(dateString);
+  return `${date.toLocaleDateString('nb-NO')} at ${date.toLocaleTimeString(
+    'nb-NO'
+  )}`;
+};
+
+export const formatPublishDateString = function (dateString: string): string {
+  const date = new Date(dateString);
+  const todaysDate = new Date();
+  const differenceMilliSeconds = todaysDate.getTime() - date.getTime();
+
+  const oneHourMilliSeconds = 1000 * 60 * 60;
+  const oneMinuteMilliSeconds = 1000 * 60;
+  const oneDayMilliSeconds = 1000 * 24 * 60 * 60;
+
+  const daysDifference = Math.floor(
+    differenceMilliSeconds / oneDayMilliSeconds
+  );
+  const hoursDifference = Math.floor(
+    differenceMilliSeconds / oneHourMilliSeconds
+  );
+  const minutesDifference = Math.floor(
+    differenceMilliSeconds / oneMinuteMilliSeconds
+  );
+
+  if (daysDifference === 0 && hoursDifference === 0) {
+    if (minutesDifference < 1) {
+      return 'Just now';
+    }
+    return `${minutesDifference} minute${minutesDifference > 1 ? 's' : ''} ago`;
+  } else if (daysDifference === 0 && hoursDifference <= 3) {
+    return `${hoursDifference} hour${hoursDifference > 1 ? 's' : ''} ago`;
+  } else if (daysDifference === 0 && date.getDay() === todaysDate.getDay()) {
+    return `Today at ${date.toLocaleTimeString('nb-NO')}`;
+  } else if (daysDifference === 1) {
+    return `Yesterday at ${date.toLocaleTimeString('nb-NO')}`;
+  }
+
+  return formatLongPublishDateString(dateString);
+};
```

Comment:
You should look into using `date-fns`. I'm hopeful that it would require less code 😅

Reply:
Awesome, will swap this out in a future PR!